core/thread/src/TImplicitMT.cxx

```diff
@@ -20,15 +20,13 @@
 
 #include "TError.h"
 #include "TThread.h"
-
+#include "ROOT/TPoolManager.hxx"
 #include <atomic>
-#include "tbb/task_scheduler_init.h"
-
 
-static tbb::task_scheduler_init &GetScheduler()
+static std::shared_ptr<ROOT::TPoolManager> &GetPoolManagerMT()
 {
-   static tbb::task_scheduler_init scheduler(tbb::task_scheduler_init::deferred);
-   return scheduler;
+   static std::shared_ptr<ROOT::TPoolManager> schedMT;
```

Comment:
Hi, Philippe.

Don't know if I'm understanding you well, but that's what we are doing in TPoolManager.cxx. This is only for TImplicitMT.

Here I need the shared pointer to be static so it's alive until I call DisableImplicitMT(). In any case this shared pointer (Specific only to ImplicitMT) points to the same TPoolManager instance as the weak pointer.